Resources/Locale/en-US/deltav/job/job-supervisors.ftl

```diff
@@ -0,0 +1 @@
+job-supervisors-command = all command officials
```

Comment:
```suggestion
job-supervisors-command = all command officials

```
Two things: Not sure how "all command officials" sounds, maybe "staff" would be better? Not sure.
This should also be checked that this appears okay in a sentence; so just load up the game on a map as an admin assistant and check the flavor text to see it looks okay in the orange starter text you get when you first join as the role

files should have a line at the end, it's good form supposedly

Reply:
Changed to all command staff, looks fine in game.